app/controllers/notes_controller.rb

```diff
@@ -111,14 +111,11 @@ def create
           @node.add_tag('event:rsvp', current_user)
           @node.add_tag('date:' + params[:date], current_user) if params[:date]
         end
-        if params[:draft] != true
```

Comment:
Oops, are we losing the draft feature here?

Reply:
Ohh I mistakenly deleted it, I will add it shortly